jupyter-js-widgets/src/widget_box.js

```diff
@@ -70,18 +70,22 @@ var ProxyView = widget.DOMWidgetView.extend({
             var that = this;
             this.child_promise = this.child_promise.then(function() {
                 return that.create_child_view(value).then(function(view) {
-                    if (that.$box.length === 0) {
-                        console.error("Widget place holder does not exist");
+                    if (that.box === undefined) {
```

Comment:
Because of the changes to the PlaceProxyModel view you made below, you'll need to change this because `[]` should also cause this conditional to evaluate to true.